build.gradle

```diff
@@ -7,6 +7,13 @@ version '1.0'
 
 allprojects {
     apply plugin: 'io.spring.dependency-management'
+    apply plugin: 'checkstyle'
+
+    checkstyle {
+        config project.resources.text.fromUri('https://raw.githubusercontent.com/UvitaTeam/code-style/master/vivy-idea-plugin/src/main/resources/META-INF/file/checkstyle.xml')
```

Comment:
why not `project.file("vivy-idea-plugin/src/main/resources/META-INF/file/checkstyle.xml")`?

Reply:
It will only work this way, if user will need to copy/add the plugin to every project, which we are trying to avoid.